styles/styles.css

```diff
@@ -129,14 +129,12 @@
   --eyebrow-font-size-s: 12px;
 
   /* heading sizes - as per interactive strategies */
-  --h1-font-size-xxxl: 44px;
-  --h1-font-size-xxl: 36px;
-  --h1-font-size-xl: 32px;
-  --h1-font-size-l: 28px;
-  --h1-font-size-m: 24px;
-  --h1-font-size-s: 20px;
-  --h1-font-size-xs: 18px;
-  --h1-font-size-xxs: 14px;
+  --heading-font-size-xxl: 36px;
+  --heading-font-size-xl: 32px;
+  --heading-font-size-l: 28px;
+  --heading-font-size-m: 24px;
+  --heading-font-size-s: 20px;
+  --heading-font-size-xs: 18px;
```

Comment:
Hi @mkbansal1 these values don't match with the ones provided by customer. I suggest using the values from the removed part with just the names changed.

Reply:
@thomsebastin  correct values are used. Removed unused variables (xxxl & xxs)

Reply:
@mkbansal1 I think there is a confusion. The variable values are correct. We just need to change the CSS to use it if it is unused.